spec/replication/replication_spec.rb

```diff
@@ -0,0 +1,166 @@
+require_relative "./replication_helper"
+
+describe "pglogical replication" do
+  let(:slave_db_name)  { "#{ActiveRecord::Base.connection.current_database}" }
```

Comment:
No need for the interpolation.  Also, it might be nicer to get it from the configuration?  `Rails.configuration.database_configuration[Rails.env]["database"]`